src/pluto/pluto.py

```diff
@@ -529,3 +564,10 @@ def create_expected(df, v1, v2):
     )
 
     create_expected(data["df_expected"], v1, v2)
+
+    # OUTLIER VALUE
+    st.header("OUTLIER ANALYSIS")
+    st.write(
+        "If nothing showed up, then it means there aren't any outlier in the current version."
```

Comment:
![Uploading Screen Shot 2022-07-21 at 11.35.08 AM.png…]()
I get this on the mapped or condo flag being set (which is maybe because those aren't ready yet?)

Reply:
thank you for flagging this. I am looking into it right now.